consensus/ibft/ibft.go

```diff
@@ -536,7 +527,7 @@ func (i *Ibft) runSyncState() {
 			// The snapshot store is currently updated for PoA inside the ProcessHeadersHook
 			callInsertBlockHook(newBlock.Number())
 
-			i.syncer.Broadcast(newBlock)
+			i.syncer.UpdateLocalLatestBlock(newBlock)
```

Comment:
Why is this a call here, and not something that is done asynchronously in the syncer itself?

It's in the spec we've shared.

All you need to do is tap into blockchain events and just fire away a pubsub message when you get a new block added event

Reply:
I was going to use Blockchain subscription in syncer. This is the left code, I guess

Reply:
Removed 65ccd2e2d1c75c12d64044b32b5540368b29b770